src/computational_graph/optimize.jl

```diff
@@ -43,21 +36,23 @@ function optimize!_v0(graphs::Union{Tuple,AbstractVector{<:AbstractGraph}}; verb
 end
 
 """
-    function optimize(graphs::Union{Tuple,AbstractVector{<:AbstractGraph}}; verbose=0, normalize=nothing)
+    function optimize(graphs::Union{Tuple,AbstractVector{<:AbstractGraph}}; level=0, verbose=0, normalize=nothing)
 
-    Optimizes a copy of given `graphs`. Removes duplicated leaves, merges chains, and merges linear combinations.
+    Optimizes a copy of given `graphs`. Removes duplicated nodes (when `level > 0`) or leaves, flattens chains, 
```

Comment:
*Removes duplicated leaves, flattens operator chains, merges linear combinations, and removes zero-valued subgraphs. When `level > 0`, also removes duplicated intermediate nodes.